### PR TITLE
Gov v3 post epoch fixes & backlog (#2279)

### PR DESCRIPTION
**Pull request by noisekit**
(cherry picked from commit c0b4f2fb570e8fdc22e57b20357beddfe18e78bc)
Re-revert of #2279

### CODE REVIEW

**Comment by moss-eth** [comment]
as per conversation in discord, there is no issue in the `verify-storage` check, nor in this update, it simply warns that there might be an issue with updating the enum, which there isn't in our specific situation. https://ethereum.stackexchange.com/questions/75959/what-size-is-an-enum-member-inside-of-a-struct

**Comment by moss-eth** [comment]
This means we can merge this, or un-revert the other commit to keep the commit history and git blame correct.

**Comment by noisekit** [comment]
Git blame is ok in this one too. Cherrypicked commit.

**Comment by noisekit** [comment]
1.  We need storage file generated and added
2. Audit status before merge

**Review by barrasso — APPROVED** [review]

File: markets/perps-market/storage.dump.json [hunk 1]
```diff
@@ -367,6 +367,36 @@
       ]
     }
   },
+  "contracts/storage/Clearinghouse.sol:Clearinghouse": {
```
[the remark this answers is not in the document]
Comment (moss-eth) [reply markets/perps-market/storage.dump.json]: done + all audit feedback is complete

**Comment by moss-eth** [comment]
> 1. We need storage file generated and added

This pr is ready to be merged minus the storage dump. The scripts in the package json are not generating a new dump.

Confirmed by @dbeal-eth, the storage checker is working as intended and this issue is a false positive.

Please let us know how to proceed @noisekit 🙏

**Comment by noisekit** [comment]
> Please let us know how to proceed @noisekit 🙏

As long as audit is OK we can merge it. Are we expected to have audit for these changes?

**Comment by moss-eth** [comment]
> As long as audit is OK we can merge it. Are we expected to have audit for these changes?

Audit is done and all changes have been made and confirmed by iosoro

**Review by moss-eth — APPROVED** [review]

**Pull request merged**